rules/sinks/storages/sqlite/go.yaml

```diff
@@ -0,0 +1,20 @@
+
+# Sink Rules for storage database Android SQLite - https://developer.android.com/
+
+sinks:
+
+  - id: Storages.Android.SQLite.Read
+    name: SQLite(Read)
+    domains:
+      - android.com
+    patterns: 
+      - "(?i)(github.com/mattn/go-sqlite).*(Query|QueryRow)"
```

Comment:
Query and QueryRow looks likea ReadAndWrite operation

Reply:
Checked again, query and QueryRow operation in SQLlite are used as read.